package.json

```diff
@@ -159,7 +159,7 @@
     "react-native-pager-view": "6.0.1",
     "react-native-qrcode-svg": "6.1.2",
     "react-native-randombytes": "3.6.1",
-    "react-native-reanimated": "2.12.0",
+    "react-native-reanimated": "2.14.4",
```

Comment:
Hmm I remember having an issue with 2.14. Did you try v3?

Reply:
Maybe they fixed the issue I was having in one of the 2.14 patches

Reply:
Hmmm let me check Android. V3 is giving me a bunch of build issues :/

Reply:
No problem, I think the 2.14 issue I had was build issues too so if it’s building it’s probably fine